### ISSUE TITLE
Apple Review UIWebView warning

### ISSUE THREAD

**Issue by RodolfoAntonici** [contentstack/contentstack-ios#6]
When I uploaded the app to AppStore I've received the following message:

```
ITMS-90809: Deprecated API Usage - Apple will stop accepting submissions of apps that use UIWebView APIs . See https://developer.apple.com/documentation/uikit/uiwebview for more information.
```

I've diagnosed that this issue come from `AFNetworking` dependence and upon further investigation I've found this topic:
https://github.com/AFNetworking/AFNetworking/issues/4428

Does ContentStack has any update about this matter?

**Comment by uttamukkoji** [comment]
Hi @RodolfoAntonici,

We are working on this to resolve issue related to AFNetworking UIWebView APIs. We will update once on this. 

Thanks for informing us about this issue.

**Comment by RodolfoAntonici** [comment]
Ok, thank you very much. There's any deadline?

**Comment by uttamukkoji** [comment]
Hi @RodolfoAntonici,

We have addressed the UIWebView warning in Pull request #7 by removing the AFNetworking dependency from SDK. You can get the updated SDK from pods [version 3.6.4](https://cocoapods.org/pods/Contentstack).

Let us know if you need anything else.
